cpp/cluster/ClusterProperties.h

```diff
@@ -32,18 +32,19 @@ class ClusterProperties
     ClusterProperties() = default;
 
     //! Compute properties of the point clusters
-    void compute(const freud::locality::NeighborQuery* nq, const unsigned int* cluster_idx);
+    void compute(const freud::locality::NeighborQuery* nq, const unsigned int* cluster_idx,
+                 const float* masses);
 
     //! Get a reference to the last computed cluster centers
     const util::ManagedArray<vec3<float>>& getClusterCenters() const
     {
         return m_cluster_centers;
     }
 
-    //! Get a reference to the last computed cluster gyration tensors
-    const util::ManagedArray<float>& getClusterGyrations() const
+    //! Get a reference to the last computed cluster moments of inertia
+    const util::ManagedArray<float>& getClusterInertiaMoments() const
     {
-        return m_cluster_gyrations;
+        return m_cluster_inertia_moments;
```

Comment:
I dislike the wording “inertia moment” — I would rename to `MomentsOfInertia` everywhere.